plot.go

```diff
@@ -87,11 +87,11 @@ func New() (*Plot, error) {
 	if err != nil {
 		return nil, err
 	}
-	x, err := makeAxis()
+	x, err := makeAxis(false)
```

Comment:
I'm changing it to 

``` go
type orientation bool
const (
    vertical orientation = true
    horizontal orientation = false
}
```
because some of the suggested names were already in use